### PR TITLE
Bones (and thus wastebread) now contain calcium

### PR DESCRIPTION
**Pull request by pjf**
#### Summary

```SUMMARY: Balance "Bones (and thus wastebread) now contain calcium"```

#### Purpose of change

Previously wastebread was calcium-free, despite being made mostly from bone meal. This change adds calcium to bones, so food that's cooked from bone meal will also contain calcium.

#### Describe the solution

I've used a value of one bone providing 96% RDA of calcium, as each bone produces 4 bone meal, and each bonemeal turned into a bonemeal tablet that provides 24% of the RDA.

With these changes, wastebread provides 12% RDA per serve.

#### Describe alternatives you've considered

- Using a different calcium value
- Adding calcium to tainted bones as well
- Just taking my damn vitamins

#### Additional context

Bone broth (which I was worried would give 960% RDA) seems to only provide trace amounts of calcium, which seems right. I haven't checked how it's made.

I had to change the type of bones to `COMESTIBLE` from `GENERIC` in order for the vitamins to show up in recipes. I don't know what effects this may have elsewhere, so would appreciate an extra set of eyes on this. I've noticed that tainted bones are already comestibles.

I haven't added calcium to tainted bones, as they're marked as not being intended as food, but I'm fine for adding a value there as well.

### CODE REVIEW

**Comment by kevingranade** [comment]
I've yet to find a source indicating that bones are a bioavailable source
of calcium.

**Comment by Reclusive-reptile** [comment]
[https://www.smithsonianmag.com/smart-news/shrew-eating-scientists-show-humans-can-digest-bone-66337580/](url)
TL;DR If a human swallows a whole shrew without chewing, some of the teeth and smaller bones don't come out the other end of you.

**Comment by kevingranade** [comment]
> bioavailable

**Comment by pjf** [comment]
I haven't deep-dived into calcium bioavailability, but a passing glance seems to suggest that [bone meal is used as a calcium supplement in livestock](https://www.feedipedia.org/node/219). Human use of bone-meal seems to mostly warn that bones tend to accumulate lead, and thus isn't recommended for civilised use.

In any case I'm not deeply attached to this change, although it feels inconsistent if survivors can craft a calcium supplement tablet from bone meal and water, but not supplement their calcium by adding bone meal to food.

On an unrelated note, I'm still learning what makes travis happy and unhappy, and hope to adjust my JSON formatting once I've managed to get the tests to run locally.

**Comment by pjf** [comment]
Page 7 of [Calcium Supplementation in Clinical Practice: A review of Forms, Doses, and Indications](https://sci-hub.tw/10.1177/0115426507022003286) has a section on calcium from bone meal. In particular:

> A 1995 randomized placebo controlled study that included 40 osteoporotic patients found that hydroxyapatite [(bone meal)] was more effective than calcium carbonate in slowing peripheral trabecular bone loss...

However:

> Calcium supplementation from bone meal has fallen out of favor. In the 1980s, calcium from bone meal was found to be contaminated with lead, arsenic, mercury, and cadmium.

and:

> Currently, insufficient evidence is available to recommend using hydroxyapatite as a form of calcium supplementation. There are many other forms of calcium that are free of contaminants and have been shown to be well absorbed and efficacious.

So it would appear that the concern is much greater around containments rather than bioavailability. Certainly in the livestock industry it seems to be extremely common to feed poultry and swine with bone meal as a calcium and phosphorous supplement.

**Comment by kevingranade** [comment]
Works for me.
